openbb_terminal/API_README.md

```diff
@@ -1,84 +1,278 @@
+df_iex = openbb.stocks.load("AAPL", source='IEXCloud')
+
+
+## From Polygon
+df_pol = openbb.stocks.load("AAPL", source='Polygon')
+
+```
+
+
+### **Easy option to switch between obtaining underlying data and charts**
+Depending on your needs, you can get the outputs in form of data (e.g. `pandas dataframe`) or charts. If the latter is what you want, simple add `chart=True` as the last parameter.
 
-Just copy-paste the code examples below into a python script or jupyter notebook, and you're ready to go.
+### 1. Getting underlying data
 
-**Basic Stock Information**\
-Prints general information about the selected stock (in this case Gamsetop)
+```
+openbb.economy.index(indices = ['sp500', 'nyse_ny', 'russell1000'], start_date = '2010-01-01')
+```
+
+<img width="575" alt="Screenshot 2022-10-04 at 00 02 23" src="https://user-images.githubusercontent.com/40023817/193700891-f4d93440-31e3-411e-9931-3a38782f68e3.png">
+
+
+
+You might be wondering how to find all the available indices. This type of information should be available in the docstring. Let's give it a try.
+
+[INSERT CHART]
+
+As mentioned in the docstring, you can access it with the following helper function.
+
+```
+openbb.economy.available_indices()
+```
+
+<img width="1078" alt="Screenshot 2022-10-04 at 00 16 36" src="https://user-images.githubusercontent.com/40023817/193702595-ecbfc84d-3ed1-4f89-9086-e975b01c4b12.png">
+
+
+
+
+### 2. Getting charts
+
+```
+openbb.economy.index(indices = ['sp500', 'nyse_ny', 'russell1000'], start_date = '2010-01-01', chart=True)
+```
+
+<img width="741" alt="Screenshot 2022-10-04 at 00 03 57" src="https://user-images.githubusercontent.com/40023817/193701075-796ffabe-3266-4d71-9a81-3042e8ca5fc8.png">
+
+
+## 2. Toolbox Layer
+In addition to financial data, you can also get access to a robust and powerful toolbox to perform analysis on different asset classes and on your portfolio.
+
+
+Imagine that you would like to leverage existing financial calculations from OpenBB and apply them on your own data. This can be done easily - OpenBB SDK's commands usually accept a `dataframe` as an input. Here you can load it your data, either via a `csv`, `excel` file, or connecting directly with an `API` or a `database`. The possibilities are endless.
+
+Let's go through an example to see how we can do it in a few simple steps. Here we shall see how to use `portfolio optimization` functionalities from OpenBB SDK.
+
+#### Step 1. Loading order book
+
+Here we will use an example orderbook for illustration purposes. You can choose to upload your own orderbook instead.
+
+```
+order_book_path = "portfolio/allocation/60_40_Portfolio.xlsx"
+tickers, categories = openbb.portfolio.po.load(excel_file = order_book_path)
+```
+
+
+#### Step 2. Optimizing portfolio
+We provide multiple portfolio optimization techniques. For instance, you can utilize basic mean-variance techniques, such as optimizing for the maximum Sharpe ratio, or minimum variance and the likes.
```

Comment:
I'd suggest not to use Max Sharpe and Min Risk because you get concentrated portfolios and ugly looking graphs. Since these techniques are not really useful, I'd suggest to stick to Risk Parity, HRP, HERC or NCO.

Reply:
You already do this later so I'd remove these ones.

Reply:
I now listed it as one of the methods we have.